marlowe-playground-client/src/MainFrame/State.purs

```diff
@@ -119,6 +124,8 @@ mkMainFrame settings =
     , eval:
         H.mkEval
           { handleQuery: handleQuery settings
+          -- FIXME try to add a handleActionWithPreventAccidentalNavigation
+          -- to avoid having to persist the boolean
```

Comment:
is this still an issue, I don't quite understand?

Reply:
It is fixed now